bugalink-backend/users/models.py

```diff
@@ -1,6 +1,20 @@
 from django.contrib.auth.models import AbstractBaseUser, BaseUserManager
+from django.core.files.storage import FileSystemStorage
 from django.db import models
 
+def user_directory_path(instance, filename):
+    # print(instance.user)
```

Comment:
Remove commented print code. You should have been noticed that bad smell by Sonarlint if you have it installed in your code editor